sdk/core/Azure.Core/tests/TestFramework.props

```diff
@@ -1,38 +0,0 @@
-﻿<Project ToolsVersion="15.0">
-  <PropertyGroup>
-    <SessionRecordsDirectory>SessionRecords</SessionRecordsDirectory>
-    <DefineConstants>$(DefineConstants);TESTFRAMEWORK</DefineConstants>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <Compile Include="$(MSBuildThisFileDirectory)\TestFramework\*.cs" Link="TestFramework\%(RecursiveDir)\%(Filename)%(Extension)" />
-    <Compile Include="$(MSBuildThisFileDirectory)\..\src\Shared\ContentTypeUtilities.cs" Link="TestFramework\ContentTypeUtilities.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)\..\src\Shared\EventSourceEventFormatting.cs" Link="TestFramework\EventSourceEventFormatting.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)\..\src\Shared\NullableAttributes.cs" Link="TestFramework\NullableAttributes.cs" Condition="'$(UseAzureCoreNullableAttributes)'!='true'" />
-    <None Update="SessionRecords\**\*.json" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Where did this go? It needs to be included into projects that use recordings?

Reply:
Missed this initially as I tested with storage which had its own directory build props file containing this line.